docs/docs/contributing/code/guidelines.mdx

```diff
@@ -0,0 +1,33 @@
+# General guidelines
+
+Here are some things to keep in mind for all types of contributions:
+
+- Follow the ["fork and pull request"](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project) workflow. Please do not try to push directly to this repo unless you are a maintainer.
+- Use the checked-in pull request template when opening pull requests. Note related issues and tag relevant maintainers.
+- Ensure your PR passes formatting, linting, and testing checks before requesting a review.
+  - If you would like comments or feedback on your current progress, you can instead mark your PR as a draft.
+  - See the sections on [Testing](/docs/contributing/code/technical_logistics#testing) and [Formatting and Linting](/docs/contributing/code/technical_logistics#formatting-and-linting) for how to run these checks locally.
+- Backwards compatibility is key. Your changes must not be breaking, except in case of critical bug and security fixes.
+- Look for duplicate PRs or issues that have already been opened before opening a new one.
+- Keep scope as isolated as possible. As a general rule, your changes should not affect more than one package at a time.
+
+## Bugfixes
+
+We welcome bugfixes to existing functionality, but ask that you communicate the issue in as much detail as you can to help
```

Comment:
We encourage and appreciate bugfixes. We ask that you:
- Explain the bug in enough detail for maintainers to be able to reproduce it.
- Avoid breaking changes if possible.
- Include unit tests that fail without the bugfix.

If you come across a bug and don't know how to fix it we ask that you open an Issue for it describing in detail the environment in which you encountered the bug.